### PR TITLE
Add Vortex Network

### PR DESCRIPTION
**Pull request by charlesvien**

### CODE REVIEW

File: servers.json [hunk 1]
```diff
@@ -1175,5 +1175,13 @@
             "zoltix.net",
             "minecraftoce.net"
         ]
+    },
+    {
+        "id": "vortexnetwork",
+        "name": "Vortex Network",
+        "addresses": [
+            "vortexnetwork.net",
+            "mc.vortexnetwork.net"
```
Comment (GrowlyX) [inline servers.json:1184]: You do not need to add sub-domains to this array, Lunar Client services already detect them.

**Review by GrowlyX — CHANGES_REQUESTED** [review]

**Review by GrowlyX — DISMISSED** [review]

**Review by ThatKawaiiSam — DISMISSED** [review]

**Comment by GrowlyX** [comment]
image
Could you resolve this? @charlesvien

**Comment by GrowlyX** [comment]
> image Could you resolve this? @charlesvien

@charlesvien

**Comment by GrowlyX** [comment]
Closing due to inactivity- feel free to create a new PR when you're back

**Pull request closed without merging**